src/objective-c/tests/BUILD

```diff
@@ -207,6 +207,10 @@ ios_unit_test(
         ":InteropTestsLocalSSL-lib",
         ":InteropTestsRemote-lib",
     ],
+    env = {
```

Comment:
where would the ${HOST_PORT_LOCAL} come from?

Reply:
Note that we currently hardcode the ports here: https://github.com/grpc/grpc/blob/9c23d7999ce827614195fe2da6653af30cb4298c/tools/internal_ci/macos/grpc_objc_bazel_test.sh#L75
and here: https://github.com/grpc/grpc/blob/9c23d7999ce827614195fe2da6653af30cb4298c/src/objective-c/tests/BUILD#L72

(And my gut feel is that you're change doesn't count on that).

Providing a short description for the PR would help make your intent clearer.